runners/direct-java/src/main/java/org/apache/beam/runners/direct/TransformExecutor.java

```diff
@@ -138,13 +141,6 @@ private void processElements(
 
         evaluator.processElement(value);
 
-        // Report the physical metrics after each element
-        MetricUpdates deltas = metricsContainer.getUpdates();
-        if (deltas != null) {
-          context.getMetrics().updatePhysical(inputBundle, deltas);
```

Comment:
That sounds fine, but as we're approaching 2 weeks on this PR's review and the code on master has diverged yet again I suggest we open this as a future ticket. We can give code references  and how suggest this change should be implemented and perhaps tag this with `starter` to encourage contributions.

@tgroh WDYT? Can you open such a ticket?